### PR TITLE
CI: add test for subproject fallbacks compilation

### PR DESCRIPTION
**Pull request by glimchb**
will rebase after #280 is merged

### CODE REVIEW

**Comment by glimchb** [comment]
@igaw this is ready for review & merge now

File: .github/workflows/meson.yml [hunk 1]
```diff
@@ -28,3 +27,23 @@ jobs:
         with:
           name: Linux_Meson_Testlog
           path: build/meson-logs/testlog.txt
+
+  meson-build-without-openssl:
```
[the remark this answers is not in the document]
Comment (glimchb) [reply .github/workflows/meson.yml:31]: we depend on `json-c.wrap` and `openssl.wrap` this CI test checks what happens when both of them are missing

[the remark this answers is not in the document]
Comment (igaw) [reply .github/workflows/meson.yml:31]: Yeah, I think the build name should be reflect what is done here. It's about testing if the fallbacks all build, right?

[the remark this answers is not in the document]
Comment (glimchb) [reply .github/workflows/meson.yml:31]: yes, correct. so `meson-build-without-fallbacks` and `meson-build-without-fallbacks` ?
if you have better naming, I don't mind, please share

[the remark this answers is not in the document]
Comment (igaw) [reply .github/workflows/meson.yml:31]: Sounds okay to me. naming is hard :)

[the remark this answers is not in the document]
Comment (igaw) [reply .github/workflows/meson.yml:31]: Given that we only build with meson these, we just might use 'build-disto', 'build-fallback' ?

[the remark this answers is not in the document]
Comment (igaw) [reply .github/workflows/meson.yml:31]: IIRC we only have access to ubuntu docker images. So we can't go crazy and do a build against some other distros.

[the remark this answers is not in the document]
Comment (glimchb) [reply .github/workflows/meson.yml:31]: ok, pushing the fix now and adjusting the commit message as well

[the remark this answers is not in the document]
Comment (igaw) [reply .github/workflows/meson.yml:31]: Though we should not go too wild here. The build resources are sponsored by github so we should be a fair user.

[the remark this answers is not in the document]
Comment (glimchb) [reply .github/workflows/meson.yml]: @igaw done. looks ok?

[the remark this answers is not in the document]
Comment (glimchb) [reply .github/workflows/meson.yml]: talking about `wild` see https://github.com/mesonbuild/meson/actions o-m-g

[the remark this answers is not in the document]
Comment (igaw) [reply .github/workflows/meson.yml]: it looks like they need to go a bit wild :D

**Pull request merged**